index.html

```diff
@@ -1,111 +1,173 @@
-                    <img src="https://assets.codepen.io/4292193/ayakoyokoya_icon.png" /> <!-- Photo -->
-                </div>
-              </div>
-
-              <div class="name">
-                <p>AYAKO YOKOYA</p> <!-- Name -->
-              </div>
-              <div class="aboutme">
-                <p>Creative Developer <span>(I wanna be...)</span></p> <!-- Profession -->
-              </div>
-
-              <div class="sns-btn-box">
-                <p>My sns accounts are...</p>
-                <a
-                  href="https://github.com/ayakoyokoya/"
-                  target="_blank"
-                  class="sns-btn"
-                >
-                  github <!-- Link to social media 1 -->
-                </a>
-                <a
-                  href="https://twitter.com/ayakoyokoya"
-                  target="_blank"
-                  class="sns-btn"
-                >
-                  twitter <!-- Link to social media 2 -->
-                </a>
-              </div>
-
-              <div class="portfolio-btn-box">
-                <p>
-                  Do you want to see error? <!-- Any text -->
-                </p>
-                <div class="portfolio">
-                  <button id="error-btn" class="error-btn">
-                    error?
-                  </button>
-                  <a
-                    href="https://ayakoyokoya.github.io/yaba-kawaii-audio-visualizer/dist/"
-                    target="_blank"
-                    class="portfolio-btn"
-                  >
-                    yes! 
-                  </a>
-                </div>
-              </div>
-            </div>
-          </div>
+  <header class="header">
+    <img src="img/Logos.svg" alt="" class="logo">
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation item"><a href="" class="nav-link">Home</a></li>
+        <li class="navigation item"><a href="" class="nav-link">Blog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Catalog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Contact Us</a></li>
+      </ul>
+    </nav>
+    <button class="search-bar">Search</button>
+    <a href="" class="discord">Discord</a>
+    <button class="connect">Connect</button>
+  </header>
+  <main class="content">
+    <section class="headline">
+      <h1>With <strong>Binabox</strong> Explore NFT collection</h1>
+      <p>We are the best way to check the rarity of NFT collection</p>
+      <ul class="achievements">
+       <li class="achievement-info">2240+</li>
+       <li class="achievement-info">Total item</li>
+       <li class="achievement-info">1000+</li>
+       <li class="achievement-info">Profile whitelisted</li>
+      </ul>
+    </section>
+    <section class="catalog">
+      <h2>Catalog</h2>
+      <a href="">See all</a>
+      <button>Right</button>
+      <button>Left</button>
+      <div class="catalog-item">
```

Comment:
@Alinchik1 "div" tag doesn't represent any semantic, we use it only as wrapper.
I think tag "article" would match it better. Like you use "article" for "Our blog" section.

Reply:
I put article catalog items as separate section and the rest can be in div?

Reply:
I wrote another one comment with the example in comment below